src/test/scala/edu/ie3/mobsim/model/ChargingBehaviorTestData.scala

```diff
@@ -18,45 +14,34 @@ trait ChargingBehaviorTestData extends TripSimulationTestData {
 
   protected def evLowSoC: ElectricVehicle = ev1.copyWith(
```

Comment:
You can turn all those `def`s back into `val`s now, should speed up tests a tiny bit :) Same for `TripSimulationTestData`